types/order.d.ts

```diff
@@ -0,0 +1,89 @@
+export interface Options {

```

Comment:
Where was this taken from?

Reply:
From here https://github.com/ripe-tech/ripe-registry-api-js/blob/e41f734fd7e4640a6756528e2ba4ac78e7a2a255/types/client.d.ts#L4

Reply:
Is that correct though? I'm seeing different stuff in yonius: https://github.com/hivesolutions/yonius/blob/master/types/api/api.d.ts

Which one is correct?

Reply:
I believe they are different things, the one on yonius is to request like get, post and others. The one on registry and here is to request to the find method of yonius (https://github.com/hivesolutions/yonius/blob/16dec63e4b6278b0708e048d7d053402f3a5e538/js/data/model.js#L282), which receive the attributes extracted there

Reply:
You're right but to be honest this should be importing the types from `yonius`. In any case, let it be as it is.